com.unity.ml-agents/Runtime/Integrations/Match3/AbstractBoard.cs

```diff
@@ -38,10 +38,10 @@ public struct BoardSize
         /// <param name="lhs"></param>
         /// <param name="rhs"></param>
         /// <returns>True if all fields are less than or equal.</returns>
-        public static bool operator <=(BoardSize lhs, BoardSize rhs)
+        public static bool operator<=(BoardSize lhs, BoardSize rhs)
         {
             return lhs.Rows <= rhs.Rows && lhs.Columns <= rhs.Columns && lhs.NumCellTypes <= rhs.NumCellTypes &&
-                   lhs.NumSpecialTypes <= rhs.NumSpecialTypes;
+                lhs.NumSpecialTypes <= rhs.NumSpecialTypes;
```

Comment:
Looks like my IDE was being helpful.

Reply:
`dotnet-format` disagrees :) I re-ran it and pushed the changes back.